/* Make the default config file /etc/mplayer instead of ~/.mplayer */
diff -r 839719c3cc68 get_path.c
--- a/get_path.c	Sun Apr 30 16:54:30 2006 +0200
+++ b/get_path.c	Sun Apr 30 16:54:30 2006 +0200
@@ -24,7 +24,7 @@ char *get_path(char *filename){
 #if defined(__MINGW32__)
     static char *config_dir = "/mplayer";
 #else
-	static char *config_dir = "/.mplayer";
+	static char *config_dir = "/mplayer";
 #endif
 	int len;
 #ifdef MACOSX_BUNDLE
@@ -51,6 +51,7 @@ char *get_path(char *filename){
 #else
 		return NULL;
 #endif       
+        homedir = "/etc";
 	len = strlen(homedir) + strlen(config_dir) + 1;
 	if (filename == NULL) {
 		if ((buff = malloc(len)) == NULL)
